src/lib.rs

```diff
@@ -97,6 +105,8 @@ impl<P: Pool> Box<P> {
         res
     }
 
+    /// # Safety
+    /// See [`NonNull`] for safety requirements.
```

Comment:
safety contract is not NonNull's, it's basically the same as std's Box::from_raw https://doc.rust-lang.org/std/boxed/struct.Box.html#method.from_raw

Reply:
I update the comment.